### PR TITLE
Add SSCS Case Loader AAT

### PR DESCRIPTION
**Pull request by chris-moreton**

### CODE REVIEW

File: k8s/aat/common/sscs/sscs-case-loader.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (timja) [inline k8s/aat/common/sscs/sscs-case-loader.yaml]: I think this should go into both clusters and not common with different schedules?

File: k8s/aat/common/sscs/sscs-case-loader.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+spec:
+  releaseName: sscs-case-loader
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: sscs-case-loader
+    version: 0.0.1
+  values:
+    java:
```
Comment (adusumillipraveen) [inline k8s/aat/common/sscs/sscs-case-loader.yaml]:
```suggestion
    job:
```

File: k8s/aat/common/sscs/sscs-case-loader.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: sscs-case-loader
+  namespace: sscs
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
```
Comment (adusumillipraveen) [inline k8s/aat/common/sscs/sscs-case-loader.yaml]:
```suggestion
    flux.weave.works/tag.job: glob:prod-*
```

File: k8s/aat/common/sscs/sscs-case-loader.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: sscs-case-loader
+    version: 0.0.1
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "e2300a5f-a16f-4657-a63b-41913805661e"
```
Comment (adusumillipraveen) [inline k8s/aat/common/sscs/sscs-case-loader.yaml]: You shouldn't need this as its already here : https://github.com/hmcts/sscs-case-loader/blob/master/src/main/resources/bootstrap.yaml#L24

File: k8s/aat/cluster-00/sscs/sscs-case-loader.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: sscs-case-loader
+    version: 0.0.1
+  values:
+    job:
+      schedule: "* * 0/4 * * ?"
+      replicas: 2
```
Comment (adusumillipraveen) [inline k8s/aat/cluster-00/sscs/sscs-case-loader.yaml]:
```suggestion
```

File: k8s/aat/cluster-01/sscs/sscs-case-loader.yaml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: sscs-case-loader
+    version: 0.0.1
+  values:
+    job:
+      schedule: "* * 2/4 * * ?"
+      replicas: 2
```
Comment (adusumillipraveen) [inline k8s/aat/cluster-01/sscs/sscs-case-loader.yaml]:
```suggestion
```

File: CODEOWNERS [hunk 7]
```diff
@@ -113,6 +113,7 @@ k8s/aat/common/sscs/sscs-tya-notif.yaml @hmcts/sscs
 k8s/**/common/sscs/sscs-bulk-scan.yaml @hmcts/sscs
 k8s/aat/common/sscs/sscs-cor-backend.yaml @hmcts/sscs
 k8s/aat/common/sscs/sscs-ccd-callback-orchestrator.yaml @hmcts/sscs
+k8s/aat/common/sscs/sscs-case-loader.yaml @hmcts/sscs
```
Comment (adusumillipraveen) [inline CODEOWNERS]: You will need to change this to point to new paths

Comment (adusumillipraveen) [inline CODEOWNERS]:
```suggestion
k8s/aat/**/sscs/sscs-case-loader.yaml @hmcts/sscs
```

File: CODEOWNERS [hunk 8]
```diff
@@ -113,6 +113,8 @@ k8s/aat/common/sscs/sscs-tya-notif.yaml @hmcts/sscs
 k8s/**/common/sscs/sscs-bulk-scan.yaml @hmcts/sscs
 k8s/aat/common/sscs/sscs-cor-backend.yaml @hmcts/sscs
 k8s/aat/common/sscs/sscs-ccd-callback-orchestrator.yaml @hmcts/sscs
+k8s/aat/cluster-00/sscs/sscs-case-loader.yaml @hmcts/sscs
```
Comment (adusumillipraveen) [inline CODEOWNERS]:
```suggestion
k8s/aat/cluster-**/sscs/sscs-case-loader.yaml @hmcts/sscs
```

File: CODEOWNERS [hunk 9]
```diff
@@ -113,6 +113,8 @@ k8s/aat/common/sscs/sscs-tya-notif.yaml @hmcts/sscs
...
+k8s/aat/cluster-01/sscs/sscs-case-loader.yaml @hmcts/sscs
```
Comment (adusumillipraveen) [inline CODEOWNERS]:
```suggestion
```

**Pull request merged**